test/helpers/integration-server.ts

```diff
@@ -0,0 +1,41 @@
+import childProcess from "child_process"
+
+import kill from "tree-kill"
+
+let serverPid = 0
+
+export const startServer = async (
+  serverStartMessage = "Server ready",
+): Promise<number> => {
```

Comment:
number is a PID (process id)?

we're moving toward branded type. you could defined pid as: `type PID = number & { readonly brand: unique symbol }`

Reply:
can do